README.md

```diff
@@ -101,51 +246,46 @@ iota.prepareTransfers(seed, transfers)
     })
 ```
 
-## Documentation
+## Supporting the project
 
-For details on all available API methods please see the [reference page](api_reference.md).
+If the IOTA JavaScript client library has been useful to you and you feel like contributing, consider posting a [bug report][new-issue], [feature request][new-issue] or a [pull request][new-pull-request].  
+We have some [basic contribution guidelines][contribution-guidelines] to keep our code base stable and consistent.
 
-Documentation of IOTA protocol and [`IRI`](https://github.com/iotaledger/iri) http API can be found on [docs.iota.works](https://docs.iota.works).
+### Cloning and bootstraping the repository on GitHub
 
-## Contributing
+1. Click the <kbd>Fork</kbd> button in the top-right corner
+2. Clone your fork and change directory into it
+3. Bootstrap your environment by doing the following:
 
-We thank everyone for their contributions. Here is quick guide to get started with iota.js monorepo:
+    ```bash
+    npm run init
+    ```
 
-### Clone and bootstrap
+This step will download all dependencies, build and link the packages together. iota.js uses [Lerna](https://lernajs.io/) to manage multiple packages. You can re-bootstrap your setup at any point with `lerna bootstrap` command.
 
-1. Fork the repo with <kbd>Fork</kbd> button at top right corner.
-2. Clone your fork locally and `cd` in it.
-3. Bootstrap your environment with:
-
-```
-npm run init
-```
+### Running tests
 
-This will install all dependencies, build and link the packages together. iota.js uses [Lerna](https://lernajs.io/) to manage multiple packages. You can re-bootstrap your setup at any point with `lerna bootstrap` command.
+Make your changes on a single package or across multiple packages and test the system by running the following from the root directory:
 
-### Run the tests
-
-Make your changes on a single or across multiple packages and test the system in integration. Run from the _root directory_:
-
-```
+```bash
 npm test
 ```
 
-To run tests of specific package just `cd` to the package directory and run `npm test` from there.
```

Comment:
It's important to have this, because full test-suite takes time. Devs might want to focus on one package at a time.